### PR TITLE
CM-56 - Course Model: Support the created_at attribute

### PR DESCRIPTION
**Pull request by mpellicer**

### CODE REVIEW

File: src/main/java/edu/ksu/canvas/model/Course.java [hunk 1]
```diff
@@ -118,6 +119,15 @@ public void setName(String name) {
         this.name = name;
     }
 
+    @CanvasField(postKey = "created_at")
```
Comment (buckett) [inline src/main/java/edu/ksu/canvas/model/Course.java:122]: It shouldn't have a postKey as we shouldn't be sending this field back to Canvas and so it's misleading.

**Comment by buckett** [comment]
Could you also add a test to CourseGetUTest test to check it's parsing it correctly.

File: src/main/java/edu/ksu/canvas/model/Course.java (hunk 1 above)
Comment (mpellicer) [reply src/main/java/edu/ksu/canvas/model/Course.java]: Sure.

**Comment by mpellicer** [comment]
> Could you also add a test to CourseGetUTest test to check it's parsing it correctly.

Of course!

**Comment by mpellicer** [comment]
Done, included the attribute in the test.

**Comment by buckett** [comment]
Magic, thanks.

**Pull request merged**